tests/core/test_subdir_data.py

```diff
@@ -203,31 +233,34 @@ def CondaSession_get(*args, **kwargs):
         fetch_repodata_remote_request(url, etag, mod_stamp)
 
 
-def test_subdir_data_prefers_conda_to_tar_bz2():
-    channel = Channel(join(dirname(__file__), "..", "data", "conda_format_repo", context.subdir))
+def test_subdir_data_prefers_conda_to_tar_bz2(platform="linux-64"):
```

Comment:
necessary to pass on platforms that don't exist in our test in-conda-source-tree repository